modules/oauth-configuring-token-inactivity-timeout.adoc

```diff
@@ -0,0 +1,99 @@
+// Module included in the following assemblies:
+//
+// * authentication/understanding-internal-oauth.adoc
+
+[id="oauth-token-inactivity-timeout_{context}"]
+= Configuring token inactivity timeout for the internal OAuth server
+
+You can configure OAuth tokens to expire after a certain amount of time that they have been inactive. By default, there is no token inactivity timeout set.
+
+[NOTE]
+====
+If token inactivity timeout is also configured in your OAuth client, that value overrides the timeout set in the internal OAuth server configuration.
+====
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have configured a password-based identity provider (IDP), such as HTPasswd or LDAP.
+
+.Procedure
+
+. Update the OAuth configuration to set a token inactivity timeout.
+
+.. Edit the OAuth configuration:
++
+[source,terminal]
+----
+$ oc edit oauth cluster
+----
++
+Add the `spec.tokenConfig.accessTokenInactivityTimeout` field and set your timeout value:
++
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: OAuth
+metadata:
+...
+spec:
+  tokenConfig:
+    accessTokenInactivityTimeout: 400s <1>
+----
+<1> Set a value with the appropriate units, for example `400s` for 400 seconds, or `30m` for 30 minutes. The minimum allowed timeout value is `300s`.
+
+.. Save the file to apply the changes.
+
+. Wait for the OAuth server Pods to restart:
```

Comment:
For these two steps, can you clarify what action they are taking when entering `oc get`? Maybe something like:

> . Check that the OAuth server pods have restarted:

(It's a little confusing that the task is to "wait" followed by running a command.)

Reply:
Good point, thanks!